### ISSUE TITLE
Cannot read property 'load' of undefined

### ISSUE THREAD

**Issue by hristijankiko** [commitizen/cz-cli#731]
I attempted to install and set up commitizen to my project for the first time, but it seems like it is throwing a strange error.

```
> npx git-cz

Cannot read property 'load' of undefined
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

This is a fresh install and no custom configuration. This is what is in my package.json config
```
"config": {
    "commitizen": {
      "path": "./node_modules/cz-conventional-changelog"
    }
  }
```

I had a look through the code and I suspect this is the line where the error is thrown
![image](https://user-images.githubusercontent.com/8568277/81026079-3c9adf80-8ebc-11ea-9567-556999b86262.png)

If anyone has any idea why it would be throwing this error please let me know.

**Comment by duartealexf** [comment]
I also just ran into this and have no idea why this is happening... I've never seen it before.
I downgraded to 3.1.2 (with cz-conventional-changelog @ 2.1.0) and it's working fine.

**Comment by Techn1x** [comment]
Having the same problem here. Upgraded to 4.1.0 and started seeing the error. Rolling back to ~4.0.5 seems to work.

**Comment by Techn1x** [comment]
To help searchability this is the error I get on 4.1.0
```
yarn git-cz        
yarn run v1.22.0
$ ./node_modules/.bin/git-cz
Cannot read property 'load' of undefined
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by Techn1x** [comment]
Looks like the configLoader was removed
https://github.com/commitizen/cz-cli/commit/6ef8afa7600da429a473290a917003177a2ec00a#diff-a7267e0501a0781d4b4f795e54454767

But in 4.1.0 there is still a reference to it
https://github.com/commitizen/cz-cli/blob/f7982d38c0882e7e91a011bfd409e2262858bfb1/src/cli/git-cz.js#L17

**Comment by hristijankiko** [comment]
@Techn1x I'm pretty sure its not the method that's undefined, its the configLoader itself.

**Comment by Wentong-Liu** [comment]
same issue here

**Comment by xmlking** [comment]
recently I am having various issues with `commitizen` I cannot commit my code anymore with `git cz`

**Comment by jimthedev** [comment]
Seems like perhaps https://github.com/commitizen/cz-cli/pull/729 modified the loader in some unexpected way @LinusU?

**Comment by jimthedev** [comment]
For anyone having problems please downgrade to 4.0.5 for now.

**Comment by LinusU** [comment]
🙈 sorry about this, investigating!

**Comment by LinusU** [comment]
This PR should fix it: #733

Will merge as soon as it is green. Super sorry for this everyone, I didn't realise that this was part of the public api.

Will also have a look at why the tests didn't catch this, it seems like they do now that a new version is published. I believe that this is because the tests install from npm, a good approach is probably to change this to `npm pack` and then `npm install /path/to/pack/file.gz` instead.

Hopefully a new version should be published in ~15 minutes!

**Comment by LinusU** [comment]
This should be fixed in version 4.1.1, super sorry for the breakage!

Also sorry for the slow response, shouldn't release and then go to bed I guess 🙈

**Comment by fahimmahmoodmir** [comment]
now I have an issue with a different error:
`The "path" argument must be of type string. Received type undefined`

![image](https://user-images.githubusercontent.com/2732245/81055617-5d1d6680-8ec9-11ea-8106-0d58288bb691.png)

**Comment by LinusU** [comment]
@fahimmahmoodmir sorry about that, it's being fixed in #735 which is currently being released.

I've written up here on why I didn't catch this when developing this, and how I will avoid releasing broken builds in the future here: https://github.com/commitizen/cz-cli/issues/734#issuecomment-624017625
